src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.DukeException;
+
+/** A subclass of Command which adds a deadline to the taskList */
+public class DeadlineCommand extends AddCommand {
+    private final String name;
+    private final String schedule;
+
+    /**
+     * Creates a DeadlineCommand.
+     *
+     * @param content The task information supplied by the user.
+     * @throws DukeException If the content has missing name or schedule or the format does not contain /by.
+     */
+    public DeadlineCommand(String content) throws DukeException {
```

Comment:
if possible, best to avoid arrowhead coding here

Reply:
Thanks for the`sharp observation. Just rectified the issue!